src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/RedirectHandler.cs

```diff
@@ -54,10 +54,19 @@ public RedirectHandler(int maxAutomaticRedirections, HttpMessageHandler initialI
                 // Clear the authorization header.
                 request.Headers.Authorization = null;
 
+                if (NetEventSource.IsEnabled)
+                {
+                    NetEventSource.Info(this, $"Redirecting from {request.RequestUri} to {redirectUri} in response to status code {(int)response.StatusCode} '{response.StatusCode}'.");
```

Comment:
>{(int)response.StatusCode} '{response.StatusCode}

Why is response.StatusCode listed twice?  You should be able to just list it as:

>{(int)response.StatusCode}

Comment applies to other code block below.

Reply:
I did this just to make it easier to read the logs. Calling ToString on HttpStatusCode returns the name of the status code, so by listing both we get logs like this:
```
Redirecting from http://localhost/307 to http://localhost/ in response to status code 307 'TemporaryRedirect'.
```
Instead of just getting the integer code:
```
Redirecting from http://localhost/307 to http://localhost/ in response to status code 307.
```
I think it's easier to read this way, but if you think it's unnecessarily verbose I can change it back.

Reply:
Ok. I understand now. I like this added clarify of the enum name as well as the enum numerical value in the trace.